src/test/java/com/artipie/api/RepositoryVerticleTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.api;
+
+import com.artipie.asto.Key;
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.asto.memory.InMemoryStorage;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpClientResponse;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import java.util.stream.Collectors;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
```

Comment:
@olenagerasimova Test is down on my pc, because HttpServer starting takes much time:
io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:8089

In my test I added such waiter: waitHttpServerUp(vertx)
https://github.com/andpopov/artipie/blob/integration_vertx-openapi/src/test/java/com/artipie/PetsopenapiTest.java

Reply:
I've seen the waiter, it causes thread blocked error... Also, in my opinion good-old thread-wait is better as it does not add any complication to the test...

Reply:
ping will allow to start test early than general sleep/wait.